Sources/HAP/Server/Device.swift

```diff
@@ -317,6 +317,22 @@ public class Device {
         return pairingState == .paired
     }
 
+    // Remove all the pairings made with this Device
+    // Can be used in the event of a stale configuration file
+    public func unpairFromAllControllers() {
+        logger.debug("Before unpair")
+        logger.debug(self.configuration)
+        logger.debug(self.config)
+        server?.stop()
```

Comment:
Why do we stop the server? It's also not clear from the method name that this happens.

Reply:
Because we need to close all open connections, to force the unpairing.

Reply:
So what's the exact use-case of this method? I'm guessing it would be some software equivalent of a hardware device's "hard reset". And so, I would expect it to also call `server?.start()`. The documentation would have to be more explicit about what this method does. E.g. should it `start()` when it wasn't already started on entry, or would that be an invalid entry state?